cookie-banner-rules-list.json

```diff
@@ -1534,23 +1511,11 @@
     },
     {
       "click": {
-        "optIn": "button#gdpr-consent-banner-accept-button",
-        "presence": "div.gdpr-consent-banner"
-      },
-      "cookies": {
-        "optIn": [
-          {
-            "name": "cto_bundle",
-            "value": "6LSRR19vaVVnVjdSWHlPZmtPSzM0bmIlMkJ4OVVFVDNDNm9MMTZ3WVYzUHhsTm1OTjZpZ0poOEl1bVNFWGp5VXB5ZSUyQnJ3RENpM25WN1QzeUxQckxjSmlyelhydmdRRzg1RlpmRXNsaE1tSGZDSXEzJTJGNG5MbEF1ZiUyQkZTVWgxbnR1bGQzRW0wdFpyakhINVM4SXpkbk9PJTJGdXQ4SE1nJTNEJTNE"
-          }
-        ],
-        "optOut": [
-          {
-            "name": "rbzid",
-            "value": "9B4wIoP8rsa+1YSv57XFo/uUSV+sZUNn9YHPdzFdQ9vj99CJquEXFnM9Icin8eAg4s61utnrutr/pbB4VnjxhdpAYo9E4+hvwG1ldnRSNgrgH67CESdXEK8ui4aWrxYINvbBwjSgrFpKg31SfphpXBJ+Rn2DUGX8HsLFm6gd0Erqfm1LGPuP0KbsVo+XacG5fDA4pTQ8CorfGyH8TTH9hIe9NozfxGveJAX7VoPFDpH7+Vmj9DlOUQ4kVR80UIhm"
-          }
-        ]
+        "optIn": ".message-component.message-button.no-children.focusable.primary.sp_choice_type_11.last-focusable-el",
+        "presence": ".message-container",
```

Comment:
This looks a lot like the other sourcepoint rules (look for ".sp_choice_type"). Can it be merged into those rules?